spec/support/database_cleaner.rb

```diff
@@ -1,16 +1,6 @@
 RSpec.configure do |config|
-  config.before(:suite) do
+  config.before(:each) do
     DatabaseCleaner.clean_with(:truncation)
     Rails.application.load_seed
```

Comment:
Loading seeds before each test will be really slow, no?
I think we should get rid of seeds completely in tests.